### Задача 1

Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

In [3]:
lambda_ = 0.95
lambda_ / 2

0.475

In [4]:
# По таблице Лапласа:
t_lambda = 1.96

In [5]:
delta = 1.96 * 16 / 256 ** (1/2)
delta

1.96

Добверительный интервал:

In [6]:
80 - delta

78.04

In [7]:
80 + delta

81.96

Ответ: (78.04, 81.96)

### Задача 2

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [8]:
import numpy as np

In [9]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [10]:
a

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [11]:
a_mean = a.mean()
a_mean

6.590000000000001

In [20]:
a_std = a.std(ddof=1)
a_std

0.4508017549014448

In [21]:
t_kr = 2.262

In [22]:
delta = t_kr * a_std / 10**(1/2)
delta

0.32246174408757394

Доверительный интервал:

In [23]:
a_mean - delta

6.267538255912426

In [24]:
a_mean + delta

6.912461744087575

Ответ: Х в интервале (6.267538255912426, 6.912461744087575) с доверительной вероятностью 0.95

### Задача 3

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

In [25]:
0.5 - 0.05

0.45

In [26]:
t_lambda = (1.64 + 1.65) / 2
t_lambda

1.645

In [29]:
U_nabl = ((17.5 - 17) / 4**(0.5)) * (100**0.5)
U_nabl

2.5

U_nabl > t_lamba => гипотеза отвергается

### Задача 4

Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [30]:
a = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [34]:
a.mean()

198.5

In [45]:
a_std = a.std(ddof=1)
a_std

4.453463071962462

In [46]:
t_kr = 3.25

In [47]:
n = 10
delta = (200 - a.mean())

In [48]:
t_nabl = delta * n**(1/2) / a_std

In [49]:
t_nabl

1.0651074037450896

In [50]:
t_nabl < t_kr

True

Гипотеза принимается